cmake/thirdparty/SetupSeracThirdParty.cmake

```diff
@@ -4,12 +4,156 @@
+        if(NOT EXISTS ${_conduit_config})
+            MESSAGE(FATAL_ERROR "Could not find Conduit cmake include file ${_conduit_config}")
+        endif()
+
+        find_package(Conduit REQUIRED
+                    NO_DEFAULT_PATH
+                    PATHS ${CONDUIT_DIR}/lib/cmake)
+    else()
+        # Allow for several different configurations of Conduit
+        find_package(Conduit CONFIG 
+            REQUIRED
+            HINTS ${CONDUIT_DIR}/cmake/conduit 
+                  ${CONDUIT_DIR}/lib/cmake/conduit
+                  ${CONDUIT_DIR}/share/cmake/conduit
+                  ${CONDUIT_DIR}/share/conduit
+                  ${CONDUIT_DIR}/cmake)
+    endif()
+
+    # Manually set includes as system includes
+    set_property(TARGET conduit::conduit 
+                 APPEND PROPERTY INTERFACE_SYSTEM_INCLUDE_DIRECTORIES
+                 "${CONDUIT_INSTALL_PREFIX}/include/")
+
+    set_property(TARGET conduit::conduit 
+                 APPEND PROPERTY INTERFACE_SYSTEM_INCLUDE_DIRECTORIES
+                 "${CONDUIT_INSTALL_PREFIX}/include/conduit/")
+else()
+    set(CONDUIT_FOUND FALSE CACHE BOOL "")
+endif()
+
+
+#------------------------------------------------------------------------------
 # MFEM
-################################
+#------------------------------------------------------------------------------
 include(cmake/thirdparty/FindMFEM.cmake)
 
+
+#------------------------------------------------------------------------------
+# Tribol
+#------------------------------------------------------------------------------
+if(TRIBOL_DIR)
+    serac_assert_is_directory(VARIABLE_NAME TRIBOL_DIR)
+
+    set(TRIBOL_INCLUDE_DIR ${TRIBOL_DIR}/include)
+
+    set(_target_file ${TRIBOL_DIR}/lib/cmake/tribol-targets.cmake)
+
+    if(NOT EXISTS ${_target_file})
+        message(FATAL_ERROR "Could not find Tribol CMake exported target file (${_target_file})")
+    endif()
+
+    include(${_target_file})
+
+    if(TARGET tribol)
+        message(STATUS "Tribol CMake exported library loaded: tribol")
+    else()
+        message(FATAL_ERROR "Could not load Tribol CMake exported library: tribol")
+    endif()
+
+    # Set include dir to system
+    set_property(TARGET tribol
+                 APPEND PROPERTY INTERFACE_SYSTEM_INCLUDE_DIRECTORIES
+                 ${TRIBOL_INCLUDE_DIR})
+
+    set(TRIBOL_FOUND TRUE CACHE BOOL "")
+else()
+    message(STATUS "Tribol support is OFF")
+    set(TRIBOL_FOUND FALSE CACHE BOOL "")
+endif()
+
+
+#------------------------------------------------------------------------------
+# Remove exported OpenMP flags because they are not language agnostic
+#------------------------------------------------------------------------------
+set(_props)
+if( ${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.13.0" )
+    list(APPEND _props INTERFACE_LINK_OPTIONS)
+endif()
+list(APPEND _props INTERFACE_COMPILE_OPTIONS)
+
+# This flag is empty due to us not enabling fortran but we need to strip it so it doesnt propagate
```

Comment:
I assume this is the black magic you were talking about.

Reply:
Oh so gross.... no better idea though.